steal.lic

```diff
@@ -114,13 +117,11 @@ class Steal
   def bin_items
     return if @bin_items.empty?
 
-    walk_to 6017
-    move('tap knocker')
-    move('north')
+    walk_to @bin_location
```

Comment:
The Riverhaven may be hard to incorporate.   I might just put all of the bins in and base everything on hometown.